concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1379,13 +1427,23 @@ jobs:
           - get: cf-manifest
             passed: ['smoke-tests', 'acceptance-tests', 'custom-acceptance-tests']
           - get: bosh-CA
+          - get: cf-secrets
+            passed: ['cf-deploy']
+
+      - task: create-temp-user
+        file: paas-cf/concourse/tasks/create_admin.yml
```

Comment:
Do the performance tests need a user with admin privileges? Can we create a less privileged user?